README.md

```diff
@@ -8,8 +24,36 @@ $ npm install
 $ npm start
 ```
 
-Currently the site is compiled to static HTML, CSS, and JS where it can run on github pages.
+## Develop Workflow
+
+After successfully building and running the server, go to localhost:8080/
+
+Now start changing files in /src and refresh localhost:8080/ You don't need to rerun the server, although it doesn't hurt to either.
+
+Some server side files need the server to be rerun with npm start.
+
+## Sitemap
+
+```
+/ -> /src/pages/home.js
+/encrypt/ -> /src/pages/encrypt/index.js
+/open-web-fellows/ -> /src/pages/open-web-fellows/overview.js
+/open-web-fellows/info/ -> /src/pages/open-web-fellows/info.js
+/open-web-fellows/fellows/ -> /src/pages/open-web-fellows/fellows.js
+```
+
+## Source Structure
+
+We use npm scripts as the entry point to build our code and run our server. All out npm scripts and npm dependencies are stored in package.json.
+
+Our client side source code lives in /src. Our HTML source is written in react and lives in /src/components/ and /src/pages/. Pages are the main index files, that are fairly specifc, and components are reusable UI elements, that are a bit more generic.
+
+We have an npm script that uses babel to compile the contents of /src from es6 to /dist as es5. These are built to run on the server and client.
+
+We use webpack to package our client side files into /public, which is handled in webpack.config.js
+
+We use react router to handle the site structure, which live in /src/avocacy-main.js and /src/encrypt-main.js
 
-### Layout
+We use node to run our server, which is handled in app.js
```

Comment:
which version of node? Node 5 has native ES6 support, Node 4 does not, so this is relatively important information